manifests/init.pp

```diff
@@ -76,14 +76,14 @@
   $service_provider    = $::vault::params::service_provider,
   $manage_service      = $::vault::params::manage_service,
   $manage_service_file = $::vault::params::manage_service_file,
-  $storage             = $::vault::params::storage,
+  Hash $storage             = $::vault::params::storage,
```

Comment:
Can these remain aligned please

Reply:
ups, this should be aligned. I will fix it

Reply:
fixed